WordPress/Classes/ViewRelated/Post/AztecPostViewController.swift

```diff
@@ -362,22 +365,48 @@ class AztecPostViewController: UIViewController {
 
 // MARK: - Actions
 extension AztecPostViewController {
-    @IBAction func cancelEditingAction() {
+
+    @IBAction func closeWasPressed() {
         cancelEditing()
     }
 
-    @IBAction func displayBlogSelector() {
+    @IBAction func blogPickerWasPressed() {
+        guard isSingleSiteMode == false else {
```

Comment:
I'm not sure why we cancel editing instead of preventing taps... This is how we do it in the hybrid editor so I'm cool with keeping the same functionality.